susemanager-utils/testing/automation/push-to-obs.sh

```diff
@@ -78,8 +80,6 @@ for p in ${PACKAGES};do
     pkg_dir=$(cat rel-eng/packages/${p} | tr -s " " | cut -d" " -f 2)
     CHOWN_CMD="${CHOWN_CMD}; chown -f -R $(id -u):$(id -g) /manager/$pkg_dir"
     CMD="/manager/susemanager-utils/testing/docker/scripts/push-to-obs.sh -d '${DESTINATIONS}' -c /tmp/.oscrc -p '${p}' ${VERBOSE} ${TEST} ${OBS_TEST_PROJECT} ${EXTRA_OPTS}"
-    # Set pipefail so the result of a pipe is different to zero if one of the commands fail.
```

Comment:
Not that matters, but I wonder why we had this comment here if we didn't use `set -o pipefail` :-D

Reply:
I moved the set to the top and forgot to move the comment on a previous commit